components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -135,6 +136,17 @@ const brave_wallet::mojom::EthereumChain kKnownNetworks[] = {
      18,
      false}};
 
+const brave_wallet::mojom::EthereumChain kExperimentalKnownNetworks[] = {
+    {brave_wallet::mojom::kFilecoinLocalhostChainId,
+     "Filecoin Ganache",
+     {kGanacheFilecoinLocalhostURL},
+     {},
+     {kGanacheFilecoinLocalhostURL},
+     "FIL",
+     "Filecoin",
+     18,
+     false}};
+
```

Comment:
Please remove these unrelated changes from this file

Reply:
done